style/app.scss

```diff
@@ -654,68 +657,110 @@ a:active
 
 #fce
 {
-  height : 40px;
-  width  : 100%;
-  border : solid 1px #000000;
+  height      : 40px;
+  width       : 140px;
+  padding-top : 20px;
 }
 
 #fcecount
 {
-  width            : 53%;
+  border-radius    : 10px;
   height           : 40px;
+  width            : 140px;
   float            : left;
-  background-color : #9c91b0;
+  background-color : #ffffff;
+  padding-top      : 8px;
   text-align       : center;
-  padding-left     : 15px;
-  padding-top      : 5px;
-  font-size        : 18px;
+  font-size        : 16px;
 }
 
-#reset
+.label-hidden
 {
-  text-align : center;
-  float      : left;
-  width      : 47%;
-  height     : 40px;
-  border     : solid 1px #000000;
-  cursor     : pointer;
+  clip        : rect(0 0 0 0);
+  clip-path   : inset(50%);
+  height      : 1px;
+  overflow    : hidden;
+  position    : absolute;
+  white-space : nowrap;
+  width       : 1px;
 }
 
-#reset:hover
+#reset-selections
```

Comment:
Make this a separate class, as you should avoid ids in CSS selectors whenever possible. Reuse existing styling from the `button` class as much as possible.

Convert the sidebar-related rules below to use classes instead of ids as well.